### PR TITLE
Fix coupling_parameter Docstring

### LINKED ISSUE #1069: collisions.py needs another review
The functionality in `plasmapy/formulary/collisions.py` went through an extensive review in PR #962.  However,  there are some inconsistencies between what is written in the docstrings and what is actually implemented in the associated function.  For example, the docstring for the `coupling_parameter` currently states

```restructuredtext
method : `str`, optional
        The method by which to compute the Coulomb logarithm.
        The default method is the classical straight-line Landau-Spitzer
        method (``"classical"`` or ``"ls"``). The other 6 supported methods
        are ``"ls_min_interp"``, ``"ls_full_interp"``, ``"ls_clamp_mininterp"``,
        ``"hls_min_interp"``, ``"hls_max_interp"``, and ``"hls_full_interp"``.
        Please refer to the docstring of `Coulomb_logarithm` for more
        information about these methods.
```

for the `method` keyword but the function only accepts values of `'classical'` and `'quantum'`.  Since so much was touched by PR #962, we need to read through each function and make sure the docstrings and contained functionality are consistent.

### PR DESCRIPTION
**Pull request by Tiger-Du**
Closes #1069.

The only inconsistency is in the docstring of coupling_parameter.

- [x] I have added a changelog entry for this pull request.

- [x] If adding new functionality, I have added tests and
      docstrings.

- [x] I have fixed any newly failing tests.

### CODE REVIEW

File: changelog/1379.doc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixes the docstring of the coupling_parameter function in plasmapy/formulary/collisions.py.
```
Comment (namurphy) [inline changelog/1379.doc.rst:1]:
```suggestion
Fixed the docstring of the `~plasmapy.formulary.collisions.coupling_parameter`
function in :file:`plasmapy/formulary/collisions.py`\ .
```
The single back ticks enclosing the whole path make our online docs link to the function, and the `:file:` role formats the text in a style to indicate that it is a file.

File: plasmapy/formulary/collisions.py [hunk 2]
```diff
@@ -1922,15 +1922,10 @@ def coupling_parameter(
...
-        default method is the classical straight-line Landau-Spitzer
-        method (``"classical"`` or ``"ls"``). The other 6 supported
-        methods are ``"ls_min_interp"``, ``"ls_full_interp"``,
-        ``"ls_clamp_mininterp"``, ``"hls_min_interp"``,
-        ``"hls_max_interp"``, and ``"hls_full_interp"``.  Please refer
-        to the docstring of
-        `~plasmapy.formulary.collisions.Coulomb_logarithm` for more
-        information about these methods.
+        The method by which to compute the coupling parameter: either
+        "classical" or "quantum". The default method is "classical".
```
Comment (namurphy) [inline plasmapy/formulary/collisions.py:1926]:
```suggestion
        ``"classical"`` or ``"quantum"``. The default method is ``"classical"``.
```
The double back ticks will make the strings show up as in-line code snippets.

**Review by namurphy — APPROVED** [review]
Looks great!  I have two minor reStructuredText formatting suggestions, and after that we can merge it pretty quick.  Thanks again!

**Comment by Tiger-Du** [comment]
Awesome! Thank you for this helpful review. The suggestions you made are committed, and I'll try to remember the formatting better in the future.

**Pull request merged**